packages/edit-site/src/components/global-styles/font-sizes/font-size.js

```diff
@@ -44,14 +44,17 @@ function FontSize() {
 		'typography.fontSizes'
 	);
 
+	const [ globalFluid ] = useGlobalSetting( 'typography.fluid' );
+
 	// Get the font sizes from the origin, default to empty array.
 	const sizes = fontSizes[ origin ] ?? [];
 
 	// Get the font size by slug.
 	const fontSize = sizes.find( ( size ) => size.slug === slug );
 
-	// Whether fluid is true or an object, set it to true, otherwise false.
-	const isFluid = !! fontSize.fluid ?? false;
+	// Whether the font size is fluid. If not defined, use the global fluid value of the theme.
+	const isFluid =
```

Comment:
Sorry for the late review. I noticed that it was already merged, but I'll comment just in case.

`settings.typography.fluid` is not necessarily a boolean value, but can be an object. That is, if the following theme.json is defined, `isFluid` will be an object:

```json
{
	"settings": {
		"typography": {
			"fluid": {
				"minFontSize": "16px"
			},
			"fontSizes": [
				{
					"name": "Theme font size 1",
					"size": "30px",
					"slug": "theme-1"
				}
			]
		}
	},
	"version": 3
}
```


We might want to change it to the following to ensure that `isFluid` is always a boolean value:

```javascript
fontSize.fluid !== undefined ? !! fontSize.fluid : !! globalFluid;
```

However, the current implementation works fine for now.

Reply:
Good point, thanks for catching.

Reply:
Thanks for noticing that @t-hamano.
I submitted the fix: https://github.com/WordPress/gutenberg/pull/64882